src/by/naumenka/lesson21/Main.java

```diff
@@ -0,0 +1,56 @@
+package by.naumenka.lesson21;
+
+//Max,Winston,male,14
+//Jaime,Blossom,female,34
+//Darya,Naumenka,female,19
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        ArrayList<Person> people = new ArrayList<>();
+        int ageCount;
+        int maleCount;
+        int femaleCount;
+        String separator = ",";
+
+        System.out.println("Введите ваш путь к файлу");
+        String path = new Scanner(System.in).nextLine(); //  src\\document.txt
+
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(path));
+
+            String document;
+            String[] infPerson;
+            while ((document = br.readLine()) != null) {
+                infPerson = document.split(separator);
+                people.add(new Person(infPerson[0], infPerson[1], infPerson[2], Integer.parseInt(infPerson[3].trim())));
```

Comment:
Trim only one parameter?))
Interesting

Reply:
Yep, and that's the reason why I have a concern.
If you start to add `.trim()`, you should do it everiwhere (also such approach is more correct and safe even if the initial document have no spaces)